src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/client/prisonersearch/PagedPrisonerResponse.kt

```diff
@@ -12,6 +13,10 @@ data class Prisoner(
   val legalStatus: LegalStatus,
   val releaseDate: LocalDate?,
   val prisonId: String?,
+  @field:JsonProperty(value = "indeterminateSentence", defaultValue = "false")
```

Comment:
These are the new fields.
In the `prisoner-search-api` response data they are called `indeterminateSentence` and `recall`, but they are booleans so for readability I have prefixed the properties in our class with `is`

It's also unclear from the `prisoner-search-api` swagger spec as to whether these fields are mandatory or not. For our purposes they need to be mandatory with a definitive `true` or `false` value, so I've taken the approach that they are probably optional in the `prisoner-search-api` response (most things are! 🙄 ), and that the absence of data means `false`